scripts/superdesk/datetime/datetime_spec.js

```diff
@@ -21,15 +22,18 @@
         });
 
         describe('datetime service', function() {
-            it('can short format datetime', inject(function(datetime, moment) {
+            it('can short format datetime', inject(function(datetime, moment, config) {
                 var now = moment();
                 var day = now.isoWeekday();
 
-                expect(now.format('hh:mm')).toBe(datetime.shortFormat(now.format()));
+                var TIME_FORMAT = config.shortTimeFormat || 'hh:mm';
+                var WEEK_FORMAT = config.shortWeekFormat || 'dddd, ' + TIME_FORMAT;
```

Comment:
imo it shouldn't use the config, it's testing the config, so you should set the value explicitly